server/schemas/item.graphql

```diff
@@ -0,0 +1,74 @@
+type Item implements Node {
+  id: ID!
+  modelId: ID!
+  model: Model!
+  values: [ItemValue!]!
+  publicVersion: String!
+  #  draftVersion: String!
+  createdAt: DateTime!
+  updatedAt: DateTime!
+}
```

Comment:
Mark "force resolver" for `lastestVersion` and `versions`. `lastestVersion` resolver should returns the latest version if the struct has already `versions`.

```suggestion
  publicVersion: String!
  createdAt: DateTime!
  updatedAt: DateTime!
  lastestVersion: ItemVersion
  versions: [ItemVersion!]!
}

type ItemVersion {
  version: String!
  parent: [String!]!
  ref: [String!]!
  fields: [ItemField!]!
}
```